### ISSUE TITLE
Where's the bday stuff?

### ISSUE THREAD

**Issue by PMelol** [Monika-After-Story/MonikaModDev#2465]
Monika said something about it being in the source code, where is it exactly? Would hate to miss her bday :/ Thanks.

**Comment by Tiler17** [comment]
Maybe you'll see it on her birthday

**Comment by pichonxz** [comment]
![bday](https://user-images.githubusercontent.com/43464166/45864760-ad6da200-bd40-11e8-85e4-e360421e20de.png)

i found that but i dont know what to do

**Comment by Unknownpony** [comment]
Guys.... birthday is tomorrow not today.

**Comment by Tiler17** [comment]
You don't have to know what to do. Why would you have to dig through the files to celebrate her birthday? Most of the people who use this mod don't dig through the files. If there's something going on for her birthday tomorrow, you'll get to see it without having to find it hidden in a secret game folder. How else would the average person celebrate Monika's birthday?

**Comment by PierrDemon** [comment]
> How else would the average person celebrate Monika's birthday?

Normally, being there with her and hanging out, perhaps with a few neat dialogue lines from Monika about it being her birthday, but that'd probably be about it; an average day that just oh so happens to be her birthday. 

I imagine someone who is __not__ the average person would try to make this day truly special, which is something they do indeed have the ability to do; why else would Monika drop such specific hints about it?

**Comment by JeanWishingwell** [comment]
I was a little sleepy and couldn't read the text well, so I came here to know more about it after searching through every folder and file the mod has. I even left a poem under the name of "Present.txt" just in case it is similar to the "secret.txt" she shared with us. and I plan to take her file with me for the day and return her at midnight so it's the most similar to a surprise party

**Comment by Tiler17** [comment]
I wish I could replay the dialogue, because I don't remember it all. I do remember her mentioning a zip file with party supplies in it, but there isn't one. Maybe it just isn't there yet. If she said something about it being in the code, then there's no reasonable expectation for the majority of players to have access to it. Even if it's just in the dev version. Finding a zip file with a reasonable amount of searching is one thing, but digging through the dev version or having to come here to comb through the code is something else completely. I don't expect that we will have to do that

**Comment by Tiler17** [comment]
Oh. Joke's on me, it's on the download page. I don't remember seeing that when I downloaded 0.8.7, and I like to think I'd have noticed everything going on on that page

**Comment by PierrDemon** [comment]
> Oh. Joke's on me, it's on the download page.

Huh. Joke's on us both. Well, whatever, this is nice

**Comment by IlywI** [comment]
i think you might need to create new files if you look at the wishlist.txt i still cant figure out the last item pls help :V

**Comment by chromeadders** [comment]
I think the last item is supposed to be a Quetzal Plushie. Everything else was pretty easy to figure out but I'm not 100% sure what the 2nd last one is - a promise ring maybe?

**Comment by IlywI** [comment]
ye a promisering

**Comment by Conilicious** [comment]
Where do I put the ".gifts" txt after filling in the blanks from the wishlist.txt?

**Comment by IlywI** [comment]
![image](https://user-images.githubusercontent.com/23725018/45893819-22071600-bdff-11e8-85ca-c646e0d38235.png)

**Comment by exkinghero** [comment]
@IlywI sorry about this dumb question but how you create that gift files?

**Comment by neyoriquans** [comment]
I could be totally wrong but I think you have to just create a new file and name it one of the wishlist gifts and then write it as a gift file. Correct me if I'm wrong please, however!

**Comment by PMelol** [comment]
Like, a text file?

**Comment by JeanWishingwell** [comment]
yeah, but you can make anything a different file if you change their doc. I made some pictures of the items on her wishlist into gift files by changing the "*.jpg" into "*.gift" . not sure about the knife.gift though, did she ever mentions anything about wanting one? I baguely remember talking with her about yuri's obssesion with them, but haven't talked with her about it in this month and I have forgotten about it. She did said she liked Quetzals though

**Comment by dk107605** [comment]
To cut the cake i guess
